### ISSUE TITLE
Not failing within timeout if expect fails or error is thrown

### ISSUE THREAD

**Issue by snowe2010** [gradecam/jasmine-co#8]
From what I understand, jasmine-co takes over the `done` and `done.fail` functions for you. So if an expect fails shouldn't it fail immediately instead of waiting for jasmine.DEFAULT_TIMEOUT_INTERVAL to be hit?

**Comment by putermancer** [comment]
Yes, it should fail immediately, and in my usage it does fail immediately when expects don't pass. Perhaps there is something interesting about the test pattern you are using that is contributing?

If you can share a simple test case illustrating your problem, it will be easier to diagnose the problem you're encountering.

**Comment by snowe2010** [comment]
I think it might be due to my use of frisby and how frisby fails tests...
